### PR TITLE
fixes this easy as shit bug

### PR DESCRIPTION
**Pull request by PKPenguin321**
Fixes https://github.com/tgstation/-tg-station/issues/15017

### CODE REVIEW

File: code/datums/spells/lichdom.dm [hunk 1]
```diff
@@ -104,7 +104,7 @@
 
 			if(!marked_item)
 				M << "<span class='caution'>You must hold an item you wish to make your phylactery...</span>"
-
+				return
 			spawn(50)
```
Comment (neersighted) [inline code/datums/spells/lichdom.dm]: While you're in here convert this to two procs and `addtimer()`

Comment (PKPenguin321) [inline code/datums/spells/lichdom.dm]: two procs of what
i dont understand anything anymore
git ruined my life
:sob:

Comment (neersighted) [inline code/datums/spells/lichdom.dm]: split everything after spawn into 'newproc'
then `addtimer(src, "newproc", 50, FALSE, M)`
just ask coderbus if you're confused

Comment (duncathan) [inline code/datums/spells/lichdom.dm]: Please don't call it newproc

Comment (neersighted) [inline code/datums/spells/lichdom.dm]: that was just an example dunc

Comment (tkdrg) [inline code/datums/spells/lichdom.dm]: I wouldn't expect PKPenguin to know how to do this, honestly.

Comment (PKPenguin321) [inline code/datums/spells/lichdom.dm]: it's true

Comment (tkdrg) [inline code/datums/spells/lichdom.dm]: Well, it can be fixed in another PR, so no big deal.

**Pull request merged**